### ISSUE TITLE
Files not matching between different users using groupfolders on the web browser.

### ISSUE THREAD

**Issue by d3al** [nextcloud/groupfolders#857]
- Nextcloud 18.03, group folder 6.0.3
- Group folders using s3 bucket

I've seen this issue for 6+ months but I'm out of ideas and it's making me worried now.  NC has been working for awhile except a few issues and this is one of them.

Users noticed that some locations in at least one group folder do not match the fileserver.  On the admin account I can see all the files on nextcloud using the browser but they can't or missing some files in that folder path.

On the fileserver I moved the files to another folder (let sync and confirmed the sync worked).  Moved the files back, sync worked but these select files are not showing up for users.  As admin I can see them but the users still don't see all of the files.  

I did groupfiles:list groupfiles:scan ID and doesn't do anything, says 0 files.  I also ran the occ file:scan -all which checks each other but adds no files added either.

I have no idea what else to do to get these files to appear in the web.  This group folder has hundreds of gigs worth of data.  Is my only option to re-sync from the fileserver?

**Comment by d3al** [comment]
The user that can see all the files correctly is apart of the admin group.
I've removed the admin group from this test user and now I'm seeing a reduced list of files in this known folder path.

So this appears to be a permissions issue, is it possible to correct it and how?

All I made was groups and assigned users to the groups.  Made a group folder and assigned the groups with their access rights.

I wonder if this is a cache issue that a cache is loaded based off a assigned groups?
